src/any_agent/frameworks/tinyagent.py

```diff
@@ -230,6 +230,7 @@ async def _run_async(self, prompt: str, **kwargs: Any) -> str | BaseModel:
                         self.completion_params["response_format"] = (
                             self.config.output_type
                         )
+                    self.completion_params["tool_choice"] = "none"
```

Comment:
is not a problem of this PR but I have just realized reviewing this. Isn't it a problem that we are directly overriding `self.completion_params` here?

Like, if there are 2 subsequent runs of the same instance and the first one reaches this point, we would be messing up the params for the next run, no?